src/cultionet/scripts/cultionet.py

```diff
@@ -211,9 +212,28 @@ def persist_dataset(args):
     config = open_config(args.config_file)
     project_path_lists = [args.project_path]
     ref_res_lists = [args.ref_res]
+    
+    region_as_list = config['regions'] is not None
+    region_as_file = config["region_id_file"] is not None
+
+    assert (
+        region_as_list or region_as_file
+    ), "Only submit region as a list or as a given file"
+
+    
+    if region_as_file:
+      file_path = config['region_id_file']
```

Comment:
Looks good -- one very minor comment. It looks like the indentation is 2 spaces instead of 4? We should probably just add an automatic formatter like Black.

Reply:
@jgrss , good catch. Let me fix that right now.
Next time I will run black before comitting 👍

Reply:
Don't worry about running `Black`. We can add it as a pre-commit.